src/components/campaignHeroSubText.tsx

```diff
@@ -0,0 +1,30 @@
+
+import {
+	Link as DSLink,
+  HorizontalRule,
+	Text,
+  Spacer,
+} from "@nypl/design-system-react-components";
+
+const CampaignHeroSubText = () => {
+   return (  
+     <>
+     {/* To Do: make the link color blue: */}
+        <Text color="ui.typography.body">
+          This site is a living database with new materials added every day,
+          featuring prints, photographs, maps, manuscripts, streaming video, and
+          more.
+        </Text>
+        <Text color="ui.typography.body">
+        Our collections include some content that may be harmful or difficult to view. <DSLink href="https://digitalcollections.nypl.org/about#nypl_harmful_content_statement">Learn more</DSLink>
+        </Text>
+        <HorizontalRule/>
+        <Text color="ui.typography.body">
+        Featured Image:
```

Comment:
Space missing at the end, add `{" "}`

Reply:
this is a live saving comment, thank you

Reply:
you're so welcome 😇